Source/ProgressionSystemRuntime/Private/Components/PSHUDComponent.cpp

```diff
@@ -109,19 +109,11 @@ void UPSHUDComponent::SavePoints(EEndGameState EndGameState)
 void UPSHUDComponent::OnGameStateChanged_Implementation(ECurrentGameState CurrentGameState)
 {
 	CurrentGameStateInternal = CurrentGameState;
```

Comment:
Please completely remove `CurrentGameStateInternal` property from HUD Component header file. It will not be hard since this property is never user anywhere else

Reply:
removed with https://github.com/h4rdmol/ProgressionSystem/pull/41/commits/ec7b010e948c6fcb826bb5d6de17672f1d24e206